sycl/include/sycl/queue.hpp

```diff
@@ -371,9 +377,15 @@ class __SYCL_EXPORT queue : public detail::OwnerLessBase<queue> {
   std::enable_if_t<std::is_invocable_r_v<void, T, handler &>, event> submit(
       T CGF,
       const detail::code_location &CodeLoc = detail::code_location::current()) {
+#ifndef __INTEL_PREVIEW_BREAKING_CHANGES
```

Comment:
nit: For `if/else` cases it's more readable to use `#ifdef <preview>/#else/#endif`, IMO.

Reply:
I don't personally have a preference. They have been flipped!